cloudbees-sdm/README.md

```diff
@@ -0,0 +1,3 @@
+# CloudBees SDM Examples
+
+Files in this folder complement mentions in CloudBees SDM product documentation, found [here](https://https://docs.cloudbees.com/docs/cloudbees-sdm/latest/). As such, they are not standalone.
```

Comment:
This is not the format you used for the other products.

Reply:
Made requested changes.